pypeclub/OpenPype
# Kitsu: Add "Show in Kitsu" ( Update for selecting Assets / Shots / SubTasks folders )

## Brief description
Add a "Show in Kitsu" action to the OpenPype launcher. 


## Description
Thanks a lot for the first version of @BigRoy : https://github.com/pypeclub/OpenPype/pull/3813 
I merged the code PR : https://github.com/pypeclub/OpenPype/pull/3813, and I added new functions.

**Current Update Info :**

When clicking on this it will allow the webbrowser to open the relevant page in Kitsu :
- **Assets** : _http://{domain}/productions/{project_id}/Assets?search=_
- **Asset_Type** : ( Props, Characters ... )  _http://{domain}/productions/{project_id}/Assets?search={Asset_Type_name}_
- **Asset** : ( myasset_001 ) _http://{domain}/productions/{project_id}/Assets/{asset_id}_
- **Asset_Task** : (modeling...)  _http://{domain}/productions/{project_id}/Assets/tasks/{task_id}_
-----
- **Shots** : _http://{domain}/productions/{project_id}/Shots?search=_
- **Sequence** : ( SQ01... ) _http://{domain}/productions/{project_id}/Shots?search={sequence_name}_
- **Shot** : (  SH01... ) _http://{domain}/productions/{project_id}/Shots/{shot_id}_
- **Shot_Task** ( Animation...)  _http://{domain}/productions/{project_id}/Shots/tasks/{task_id}_

## Testing notes:

1. Enable kitsu module and link with project
2. Sync project
3. In OpenPype launcher go into the project, you should see a "Show in Kitsu" button
4. Select your asset or folder and click on "Show in Kitsu"
